drivers/bus/acpi_new/main.c

```diff
@@ -0,0 +1,16 @@
+#include "precomp.h"
+//#define NDEBUG
+#include <debug.h>
+
+CODE_SEG("INIT")
+NTSTATUS
+NTAPI
+DriverEntry (
+    PDRIVER_OBJECT  DriverObject,
+    PUNICODE_STRING RegistryPath
+    )
```

Comment:
Please use SAL2 annotations for the new code.
```suggestion
DriverEntry(
    _In_ PDRIVER_OBJECT DriverObject,
    _In_ PUNICODE_STRING RegistryPath)
```